lang/es_ES.json

```diff
@@ -303,11 +286,63 @@
     "Title": "Título",
     "Select a game": "Selecciona un juego",
     "Add to Want to Play Games": "Añadir a juegos que quiero jugar",
-    "Remove from Want to Play Games": "Eliminar de juegos que quiero jugar",
+    "Remove from Want to Play Games": "Quitar de juegos que quiero jugar",
     "Ascending (A - Z)": "Ascendente (A - Z)",
     "Descending (Z - A)": "Descendente (Z - A)",
-    "Earliest": "Más antigua",
+    "Earliest": "Más antiguo",
     "Latest": "Más reciente",
     "Less": "Menos",
-    "Both": "Ambos"
+    "Both": "Ambos",
+
```

Comment:
Jump line

Reply:
FWIW I see this as well. The line count in this file should match en_US in `master`.

![Screenshot 2024-10-29 at 12 03 03 PM](https://github.com/user-attachments/assets/452f3f31-58bd-45a8-bb9f-a792acf0b4d5)

Reply:
I'm checking my last commits because I see it's not sync. :\

Reply:
![image](https://github.com/user-attachments/assets/af5e1a56-2503-49ad-9728-ce90fcf3de58)

It's seems a GitHub issue or something... Now it's showing the good one.